topics/w4-visual-debugging.md

```diff
@@ -1,11 +1,5 @@
-# W4 - Visual Debugging
-
-### Reading List
-
-#### Required Reading
-* [PythonTutor](https://pythontutor.com/): Guo, P. J. (2013, March). Online python tutor: embeddable web-based program visualization for cs education. In Proceeding of the 44th ACM technical symposium on Computer science education (pp. 579-584).
-
-#### Optional Reading
-* [Ply:](https://github.com/sliminality/ply) Lim, S., Hibschman, J., Zhang, H., & O'Rourke, E. (2018, October). Ply: A visual web inspector for learning from professional webpages. In Proceedings of the 31st Annual ACM Symposium on User Interface Software and Technology (pp. 991-1002).
-* [Log-it](https://creativity.ucsd.edu/logit): Jiang, P., Sun, F., & Xia, H. (2023, April). Log-it: Supporting Programming with Interactive, Contextual, Structured, and Visual Logs. In Proceedings of the 2023 CHI Conference on Human Factors in Computing Systems (pp. 1-16).
-
+# Leveraging Visual Debugging for Programming Education
+### The Challenges of Teaching Program Execution
+### What is Python Tutor?
+### How Python Tutor works
```

Comment:
Since Python tutor has been around for a while, it would be great to talk about what has been done in between